sdk/src/main/java/org/coralibre/android/sdk/internal/TracingService.java

```diff
@@ -57,7 +57,8 @@
     // The source of these Values is this a description on Googles API description:
     // https://developers.google.com/android/exposure-notifications/ble-attenuation-overview
     public static final long SCAN_DURATION = 4 * 1000; // unit in milliseconds
-    public static final long SCAN_INTERVAL = 5 * 60 * 1000; // unit in milliseconds
+    public static final long SCAN_INTERVAL_SECONDS = 5 * 60; // unit in milliseconds
```

Comment:
unit in seconds, not milliseconds

Reply:
I changed the names and removed the comments, ty